src/shared/services/jwt/jwt.service.ts

```diff
@@ -0,0 +1,54 @@
+import jwt from 'jsonwebtoken';
+import { DateTime } from 'luxon';
+
+export interface IGenerateTokens_Props {
+  jwtSecret: string;
+  sessionKey: string;
+  accessLifetime: string;
+  refreshLifetime: string;
+}
+
+export interface IGenerateTokens_Return {
+  accessToken: string;
+  refreshToken: string;
+  accessTokenExpiresIn: string;
+  refreshTokenExpiresIn: string;
+}
+
+export const generateTokens = async ({
+  sessionKey,
+  jwtSecret,
+  accessLifetime,
+  refreshLifetime,
+}: IGenerateTokens_Props): Promise<IGenerateTokens_Return> => {
+  const date = DateTime.utc();
+
+  let accessToken = '';
+  let refreshToken = '';
+
+  const createToken = async (tokenLifetime: string) => {
```

Comment:
Why is this function here? It will be re-created each time the outer function runs. Move it outwards.